src/gg.regression.unity.bots/Tests/TestFramework/Scripts/RegressionGames.TestFramework.asmdef

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "name": "RegressionGames.TestFramework",
+  "references": [
+    "RegressionGames",
+    "UnityEngine.TestRunner",
+    "UnityEditor.TestRunner"
```

Comment:
Does this `UnityEditor` reference work when we run our build tests in docker ?

Reply:
Yes, you can see it passing in this PR as well above. I also verified that the tests work when run in standalone mode. It is standard for both play mode and edit mode tests to refer to both the `UnityEngine.TestRunner` and `UnityEditor.TestRunner` assemblies (this is the same practice employed by other test framwork libraries too, such as InputSystem.TestFramework).